ORB Algorithm/ORB_Algorithm.py

```diff
@@ -0,0 +1,60 @@
+import cv2
+import numpy as np
+
+# Load the image
+path=input('Enter the path of the image: ')
+image = cv2.imread(path)
+path2=input('Enter the path for testing image: ')
+test_image=cv2.imread(path2)
+
+#Resizing the image
+image=cv2.resize(image,(600,600))
+test_image=cv2.resize(test_image,(600,600))
+
+# Convert the image to gray scale
+gray = cv2.cvtColor(image, cv2.COLOR_RGB2GRAY)
+test_gray = cv2.cvtColor(test_image, cv2.COLOR_RGB2GRAY)
+
+#Display the given and test image
+cv2.imshow('output',image)
```

Comment:
```suggestion
image_stack = np.concatenate((image, test_image), axis=1)
cv2.imshow('image VS test_image', image_stack)
```
Display the images side by side instead of one by one